File: FootballExplorer/app/src/main/java/fr/a42consulting/footballexplorer/View/IPlayersListView.java
package fr.a42consulting.footballexplorer.View;

import java.util.ArrayList;

import fr.a42consulting.footballexplorer.Model.PlayersListPlayer;

public interface IPlayersListView {
    void onRequestResult(ArrayList<PlayersListPlayer> players);
    void onMessage(String message);
}

File: FootballExplorer_kotlin/app/src/main/java/fr/a42consulting/footballexplorer/view/IPlayersListView.kt
package fr.a42consulting.footballexplorer.view

import java.util.ArrayList

import fr.a42consulting.footballexplorer.model.PlayersListPlayer

interface IPlayersListView {
    fun onRequestResult(players: ArrayList<PlayersListPlayer>)
    fun onMessage(message: String)
    fun onRequestFailed()
}
